src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -744,32 +744,32 @@ export const AutomatedAnalysisCard: React.FC<AutomatedAnalysisCardProps> = (prop
   }, [usingArchivedReport, usingCachedReport, report, isLoading, errorMessage]);
 
   return (
-    <ResizableCard
+    <DashboardCard
       dashboardId={props.dashboardId}
       cardSizes={AutomatedAnalysisCardSizes}
-      className="dashboard-card"
       id="automated-analysis-card"
-      isRounded
       isCompact
       isExpanded={isCardExpanded}
+      cardHeader={
+        <CardHeader
+          onExpand={onCardExpand}
+          toggleButtonProps={{
+            id: 'automated-analysis-toggle-details',
+            'aria-label': 'Details',
+            'aria-labelledby': 'automated-analysis-card-title toggle-details',
+            'aria-expanded': isCardExpanded,
+          }}
+        >
+          <CardActions>{...props.actions || []}</CardActions>
+          <Level hasGutter>
```

Comment:
I think we should apply some truncating styles for label texts on header.

![Screenshot from 2023-01-26 12-17-54](https://user-images.githubusercontent.com/68053619/214906935-fafb4043-0cb6-4b4c-9ea5-8e823652afb1.png)

Reply:
On this note, seems like this happens because the card goes into `scroll mode` when in small spaces.

Probably because of this css class

```
.dashboard-card {
  width: 100%;
  overflow: auto;
}
```

Reply:
Been playing around with this, there seems not to be a neat way to fix unless [Container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) is supported.

I think the best way now is to keep it scrollable :D Or maybe other ways^^

Reply:
I'm not sure about other ways for now, scrollable seems to be the best fit for now as a user would probably always have the choice to reduce their viewport tiny, but I'm not sure why they would... 

As for the automated analysis card itself, maybe the best thing to do is to have the card itself to handle its own overflow style, not too sure.